### ISSUE TITLE
Typing

### ISSUE THREAD

**Issue by cblegare** [talkiq/gcloud-aio#212]
It would be nice to have type hints or stubs for this library!

Mypy error: 

```
error: Skipping analyzing 'gcloud.aio.pubsub': found module but no type hints or library stubs
```

**Comment by TheKevJames** [comment]
Huh, that's odd -- our pubsub library (and the others) is [pretty much fully type hinted](https://github.com/talkiq/gcloud-aio/blob/master/pubsub/gcloud/aio/pubsub/subscriber_client.py) (though we are not currently running `mypy` as a linter due to reasons which should have recently become unblocked, so I'd be 100% in favour of starting to do that and fixing any issues which might pop up!!). I wonder if this has to do with how we've set up the shared namespaces across the multiple packages -- it could be the case we got something wrong there, since I've never really tried to do this before!

That said, we make use of this package internally and haven't run into this issue -- mypy seems to be able to properly trace type hints through the library in our case. Can you share anything about your configuration which might help us debug what's going wrong here?

**Comment by cblegare** [comment]
Maybe simply including an empty `py.typed` file would to the trick! 

see https://www.python.org/dev/peps/pep-0561/#packaging-type-information

**Comment by TheKevJames** [comment]
This seems to be working now with the above `py.typed` changes :)
